plugins/kubernetes/app/models/kubernetes/resource.rb

```diff
@@ -380,11 +380,16 @@ def pods_count
       end
 
       def wait_for_termination_of_all_pods
-        30.times do
+        60.times do
```

Comment:
How will it work with RollingUpdate of the DaemonSet?

Reply:
I'll look into the whole RollingUpdate/OnDelete business next, hoping this is a good quick-fix for the meantime